tests/unit/test_doubles_test.sh

```diff
@@ -24,6 +24,25 @@ EOF
   assert_empty "$(assert_successful_code "$(code)")"
 }
 
+function test_successful_unmock_mock() {
+  mock ps<<EOF
+PID TTY          TIME CMD
+13525 pts/7    00:00:01 bash
+24162 pts/7    00:00:00 ps
+EOF
+
+  assert_is_mock ps
+  unmock ps
+  assert_is_not_mock ps
+}
+
+function test_successful_unmock_spy() {
```

Comment:
where are the unsuccessful test for this?

Reply:
Added